release-testing/test

```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env bash
-set -euxo pipefail
+
+# set up logging
+rm -f info-log.txt debug-log.txt error-log.txt
+touch info-log.txt debug-log.txt error-log.txt
+tail -f info-log.txt &
+
+exec 1>>debug-log.txt
```

Comment:
Needs some explanation comments - I don't think most people worked w/ the FDs with numbers over 2 so this part will be pretty rough for them.

Reply:
Agreed